src/lib/abacus/index.ts

```diff
@@ -111,6 +111,8 @@ class AbacusStateManager {
 
     const appConfigs = AbacusAppConfig.Companion.forWebAppWithIsolatedMargins;
     appConfigs.onboardingConfigs.squidVersion = OnboardingConfig.SquidVersion.V2;
+    appConfigs.staticTyping = import.meta.env.MODE == 'development'
```

Comment:
I think we can probably update this to `import.meta.env.MODE !== 'production'` so that this can be live on testnet as well. lmk if you agree